docs/_includes/api/active_tasks.html

```diff
@@ -0,0 +1,45 @@
+{% include anchor.html edit="true" title="List active tasks" hash="active_tasks" %}
+
+{% highlight js %}
+PouchDB.activeTasks.list()
+{% endhighlight %}
+
+List all active database tasks. There are three types of internal tasks: `database_compaction`, `view_indexing`, and `replication`. PouchDB will report progress of these tasks to the active tasks API and remove tasks as soon as they are completed or have failed.
+
+#### Example Usage:
+
+{% highlight js %}
+var tasks = PouchDB.activeTasks.list()
+{% endhighlight %}
+
+#### Example Result:
+
+{% highlight js %}
+{
```

Comment:
this is not a list?

Reply:
it's a list and shouldn't be an array :-) forgot the brackets.